source/mainnet/net/guides/create-proofs.rst

```diff
@@ -0,0 +1,170 @@
+
+General rules
+=============
+
+For the dApp or service developer there are some general rules about proofs that you have to follow.
+
+- There is no limit to the amount of attributes that can be revealed.
+- An attribute can only be used in one proof at a time.
+
+The attributes that can be revealed are:
+
+- First name
+- Last name
+- Sex
+- Date of birth
+- Country of residence
+- Country of nationality
+- ID document type
+- ID document number
+- ID document issuer
+- ID valid from
+- ID valid to
+- National ID number
+- Tax ID number
+
+Asking a user to reveal attributes
+==================================
+
+You can construct a reveal statement in which the wallet owner must consent to reveal one or more specific attributes.
+
+In the example below, the wallet owner's country of residence is requested to be revealed.
+
+.. code-block:: console
+
+    [
+      {
+        "type": "RevealAttribute",
+        "attributeTag": "countryOfResidence"
+      }
+    ]
+
+Unless it is absolutely critical to know a specific attribute, it is good policy to use the proof types below instead to determine a user's eligibility for your app instead of requesting that they reveal information to you.
+
+Range proofs
+============
+
+:ref:`Range proofs<glossary-range-proofs>` allow you to present a range of values to the user's wallet to determine whether the user meets the required attribute. For example, you might want to know if a user over 18 years of age but under 30 years of age. You can present this as a range proof to be verified by the wallet.
+
+Relevant attributes for range proofs are:
+
+- Date of birth
+- ID valid to
+- ID valid from
+
+.. Note::
+
+  Age proofs are a common abstraction on date of birth proofs. There are helper functions in the SDK(link?) to aid in constructing these.
+
+Structure a range proof
+-----------------------
+
+In the example below, you see a range proof constructed to check that wallet owner is between 25 and 65 years old (as of 2022-12-12).
+
+.. code-block:: console
+
+    {
+      "challenge": "a54bc4116655d247fa625d98f768d4d81e55ffe26ac6bab259bad5895d49ae00",
+      "statement": [
+        {
+          "type": "AttributeInRange",
+          "attributeTag": "dob",
+          "lower": "19571212",
+          "upper": "19971212"
+        }
+      ]
+    }
+
+Membership proofs
+=================
+
+:ref:`Membership proofs<glossary-membership-proof>` allow you to present a list of values to the user's wallet to determine whether the user has one of the required attributes in the list. For example, you might want to know if a user is a resident of the EU. You can present this as a membership proof to be verified by the wallet.
+
+Relevant attributes for membership proofs are:
+
+- Country of residence
+- Country of nationality
+- Identity document type
+- Identity document issuer
+
+.. Note::
+
+  Country of nationality has helper functions in the SDK(link?) to aid in constructing statements.
```

Comment:
If it's the EU-membership helper you mean, here is a link to the docs for the JS SDK:

https://github.com/Concordium/concordium-node-sdk-js/tree/main/packages/common#eu-membership